### PR TITLE
Add compatible-with-quarkus-core to quarkiverse-systemd-notify.yaml

### PR DESCRIPTION
**Pull request by Eng-Fouad**

### CODE REVIEW

File: extensions/quarkiverse-systemd-notify.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 group-id: "io.quarkiverse.systemd.notify"
 artifact-id: "quarkus-systemd-notify"
 versions:
-- "0.2.0"
-- "0.1.0"
+  - "0.2.0":
+      compatible-with-quarkus-core:
+        - "2.10.0.Final"
```
Comment (gastaldi) [inline extensions/quarkiverse-systemd-notify.yaml:7]: Actually we still haven't implemented minimum version comparison in the compatibility list here. 

In your case I think the best is to exclude the 0.1.0 version (by moving 0.1.0 to the exclude-versions) and mark it as compatible with 2.14.3.Final

Comment (Eng-Fouad) [reply extensions/quarkiverse-systemd-notify.yaml:7]: so, the file should be like this, right?

```
---
group-id: "io.quarkiverse.systemd.notify"
artifact-id: "quarkus-systemd-notify"
versions:
- "0.2.0"
    compatible-with-quarkus-core:
        - "2.14.3.Final"
exclude-versions: ["0.1.0"]
```

Comment (gastaldi) [reply extensions/quarkiverse-systemd-notify.yaml]: Yeap, that looks good

File: extensions/quarkiverse-systemd-notify.yaml [hunk 2]
```diff
@@ -3,5 +3,7 @@ group-id: "io.quarkiverse.systemd.notify"
 artifact-id: "quarkus-systemd-notify"
 versions:
 - "0.2.0"
```
Comment (gastaldi) [inline extensions/quarkiverse-systemd-notify.yaml:5]: Add a `:` in the end of this line

**Review by gastaldi — APPROVED** [review]

**Pull request merged**